### PR TITLE
Store fixtures locally and remove module-under-test from Puppetfile

### PR DESCRIPTION
**Pull request by mlafeldt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # Bundler
 /.bundle/
-/.gems/
```
Comment (mlafeldt) [inline .gitignore:3]: Please use `vendor/gems` instead (bundler convention).

**Comment by mlafeldt** [comment]
@zined This updates the beanstalkd module to use the "final" Vagrant testing stuff from puppet-skeleton.

**Comment by nightlyone** [comment]
vendor/gems is the "right way to do it (TM)", but puppet-lint works better with .gems.

So we can choose:
a) have no puppet-lint while keeping gems per module
b) have no different gems per module

@zined , @mlafeldt discuss

**Comment by mlafeldt** [comment]
@nightlyone @zined 

This is what `rake lint` currently does: https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L122-L125

This is what puppet-lint's Rake task does:
https://github.com/rodjek/puppet-lint/blob/master/lib/puppet-lint/tasks/puppet-lint.rb

Not sure which one is worse. :/

I suggest to use `.bundle` for now (already in the .gitignore list), although puppet-lint will still try to check everything in `fixtures`.

Anyway, what I'm doing in https://github.com/Jimdo/puppet-skeleton/pull/2 will help us with puppet-lint as well. Stay tuned. :)

**Comment by mlafeldt** [comment]
Update: The puppet-lint Rake task actually can be configured: https://github.com/rodjek/puppet-lint#rake-task-1

Will take another look.

**Comment by nightlyone** [comment]
``` ruby
PuppetLint.configuration.ignore_paths = ["vendor/**/*.pp", "fixtures/**/*.pp"]
```

looks like a possible configuration here

**Comment by mlafeldt** [comment]
Yes, that's what I figured out as well.

**Comment by mlafeldt** [comment]
Re-added `.gems` to `.gitignore` for now.

With commit https://github.com/mlafeldt/puppet-skeleton/commit/0a44d2ac5d6d3f7fe94478137f6af493bdfa4f06, puppet-lint will properly ignore `vendor` and `fixtures`.

Feel free to merge.

**Comment by mlafeldt** [comment]
Hasn't been merged and is kind of outdated now. Closing.

**Pull request closed without merging**